lib/datastore.js

```diff
@@ -5,7 +5,52 @@
  */
 
 const UUID = require("uuid"),
-      instance = require("./util/instance");
+      errors = require("./util/errors"),
+      instance = require("./util/instance"),
+      jose = require("node-jose");
+
+const DEFAULT_PROMPTS = {
+  unlock: async (request) => {
+    if (1 < request.attempts) {
```

Comment:
Nit: `if (request.attempts > 1)` would probably be clearer, but I don't know if we have a specific rule about that.